packages/shared/src/components/RenderMarkdown.tsx

```diff
@@ -128,6 +128,9 @@ const RenderMarkdown = ({
       linkTarget="_blank"
       remarkPlugins={plugins}
       components={{
+        ol({ children }) {
+          return <ol>{children}</ol>;
+        },
```

Comment:
Why is this needed, does it not already return it like this?

Reply:
it does return the exact same thing without this, but for some reason explicitly setting this fixes the issue -- my hunch is that it has to do with re-renders of these components during streaming, but I could not tell you exactly why this solves it...

Reply:
No idea why it would work but if it works let's role 🚀